examples/linear_model/plot_ridge_coeffs.py

```diff
@@ -0,0 +1,87 @@
+"""
+==============================================================
+Plot Ridge coefficients as a function of the regularization II
+==============================================================
+
+.. currentmodule:: sklearn.linear_model
+
+:class:`Ridge` Regression is the estimator used in this example.
+Each color in the left plot represents a different feature of the
+coefficient vector, and this is displayed as a function of the
+regularization parameter.
+
+In this example the dependent variable Y is set as a function
+of the input features: Y = Xa + bias. The coefficient vector a is
+randomly sampled from a normal distribution, as well as the bias
+term.
+
+As alpha tends toward zero the coefficients found by Ridge
+regression stabilize towards the randomly sampled vector a.
+For big alpha (strong regularisation) the coefficients
+are smaller (eventually converging at 0) leading to a
+simpler and biased solution.
+These dependencies can be observed on the left plot.
+
+The right plot shows the mean squared error between the
+coefficients found by the model and the chosen vector a.
+Less regularised models find the coefficients without
+trouble (error is equal to 0), stronger regularised
+models increase the error.
+"""
+
+# Author: Kornel Kielczewski -- <kornel.k@plusnet.pl>
+
+print(__doc__)
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.linear_model import Ridge
+from matplotlib import pyplot
+from sklearn.metrics import mean_squared_error
+
+
+def generate_test_data(N, p):
+    """Generate N samples of p dimensions with fixed bias"""
+    bias = np.random.randn()
+    return make_regression(N, p, coef=True, bias=bias)
```

Comment:
Ok, what you suggest is to make this deterministic? I'll change that.

What I had in mind was to genreate random input to demonstrate that the behaviour does not depend on the particular values of the input data